packages/core-utils/src/route.ts

```diff
@@ -151,9 +160,13 @@ function makeTransitOperatorComparator(transitOperators) {
  * Gets the desired sort values according to an optional getter function. If the
  * getter function is not defined, the original sort values are returned.
  */
-function getSortValues(getterFn, a, b) {
-  let aVal;
-  let bVal;
+function getSortValues(
+  getterFn: (item: unknown) => unknown,
```

Comment:
Wow this needs TypeScript FIXME comments...

Reply:
I can add a comment but they were made `unknown` explicitly. This method does not know the contents of what it is receiving as it's designed to accept anything